### PR TITLE
Fix feature gate for hugepage

### PR DESCRIPTION
**Pull request by tengqm**
The text says "disable" because the gate is in Beta, but the example says "enable". That is awkward.

### CODE REVIEW

File: content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 1]
```diff
@@ -104,17 +104,18 @@ spec:
...
-  disabled with the `HugePageStorageMediumSize` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) on the {{<
-glossary_tooltip text="kubelet" term_id="kubelet" >}} and {{<
-glossary_tooltip text="kube-apiserver"
-term_id="kube-apiserver" >}} (`--feature-gates=HugePageStorageMediumSize=true`).
+  disabled with the `HugePageStorageMediumSize`
+  [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  on the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and
+  {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+  (`--feature-gates=HugePageStorageMediumSize=false`).
```
Comment (tengqm) [inline content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md:121]: To reviewers: this change from `true` to `false` is  the main purpose of this PR. All other changes are just indentation and wrapping of long lines.

**Review by sftim — COMMENTED** [review]
@tengqm, can you separate out the whitespace changes into their own commit? Otherwise I think this is good.

**Comment by tengqm** [comment]
@sftim yes I can rework this PR to two commits if that makes you happy. The git tool doesn't have a good resolution to highlight line wraps so the review is always a pain. That is the reason why I have left a note to reviewers, and that is the reason why I don't like unnecessary text changes in PRs.

**Pull request merged**